package.json

```diff
@@ -28,9 +28,12 @@
     "@blackbaud/auth-client": "^2.7.0",
     "@skyux/config": "^3.0.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "jwt-decode": "2.2.0"
+  },
   "devDependencies": {
     "@blackbaud/skyux": "2.30.0",
```

Comment:
Okay if we bump the version of `@blackbaud/skyux` to `2.49.1` to make sure it jives well with the latest stuff?

Reply:
Updated.